### PR TITLE
Source culture in contact info form, refs #11358

### PR DESCRIPTION
**Pull request by mcantelon**
When editing an institution's contact information, show source culture values
of translatable fields.

### CODE REVIEW

File: apps/qubit/modules/contactinformation/actions/indexAction.class.php [hunk 1]
```diff
@@ -108,6 +108,30 @@ public function execute($request)
...
 
+    // Add source culture information if necessary
+    $culture = sfContext::getInstance()->user->getCulture();
+
+    if ($culture != $this->resource->sourceCulture)
+    {
+      $sourceCultureData = array('fields' => array());
+
+      // Add source culture direction, if necessary
+      $sourceCultureInfo = sfCultureInfo::getInstance($this->resource->sourceCulture);
```
Comment (jraddaoui) [inline apps/qubit/modules/contactinformation/actions/indexAction.class.php]: Is this really necessary, why can we get the source culture value in here like it's being done bellow? Could you explain a little why is this needed in a comment?

**Review by jraddaoui — CHANGES_REQUESTED** [review]
Looks great @mcantelon! Just a question and suggestion to add some comments.

File: apps/qubit/modules/contactinformation/actions/indexAction.class.php (hunk 1 above)
Comment (mcantelon) [reply apps/qubit/modules/contactinformation/actions/indexAction.class.php]: It makes the below line below 40 characters shorter, but I'll get rid of it as it does make the code a bit harder to follow.

Comment (mcantelon) [reply apps/qubit/modules/contactinformation/actions/indexAction.class.php]: Oh, I reference the culture info on line 122 as well... I've expanded on the comment and changed the code so I just take store the source culture direction in a convenience variable, rather than general source culture info, to make the purpose of the convenience variable more explicit.

Comment (mcantelon) [reply apps/qubit/modules/contactinformation/actions/indexAction.class.php]: (I've expanded the comment a bit to clarify function of the direction data a bit. Thanks!)

**Review by jraddaoui — APPROVED** [review]
That's perfect. Thanks @mcantelon!

**Comment by mcantelon** [comment]
NP! Thanks!
